src/indra_cogex/sources/indra_ontology/__init__.py

```diff
@@ -36,12 +37,32 @@ def __init__(self, ontology: Optional[IndraOntology] = None):
     def get_nodes(self):  # noqa:D102
         for node, data in self.ontology.nodes(data=True):
             db_ns, db_id = self.ontology.get_ns_id(node)
-            yield Node(db_ns, db_id, ["BioEntity"], data)
+            name = self.ontology.get_name(db_ns, db_id)
+            node_data = {"name": name}
+            parsed_data = _get_data(data)
+            node_data.update(parsed_data)
```

Comment:
`data` for a IndraOntology node already contains a `name` property so doing a `get_name` is just retrieving what is already in there.

Reply:
Resolved in https://github.com/gyorilab/indra_cogex/pull/149/commits/786f0cdd960559d88fdeb5295db94013474db1d6